content/en/docs/instrumentation/python/automatic/example.md

```diff
@@ -287,6 +307,47 @@ If those headers are available, they will be included in your span:
 }
 ```
 
+### Execute a programmatically instrumented server
+
+It is also possible to use the instrumentation libraries (such as
+`opentelemetry-instrumentation-flask`) by themselves which may have an advantage
+of customizing options. However, by choosing to do this it means you forego
+using auto-instrumentation by starting your application with
+`opentelemetry-instrument` as this is mutually exclusive.
+
+Execute the server just like you would do for manual instrumentation, in two
+separate consoles, one to run each of the scripts that make up this example:
+
+```console
+$ source auto_instrumentation/bin/activate
+$ python server_programmatic.py
+```
+
+```console
+$ source auto_instrumentation/bin/activate
+$ python client.py testing
+```
+
+The results should be the same as running with manual instrumentation.
+
+#### Using programmatic instrumentation features
```

Comment:
This section should be nested under `Execute` rather than `Configure`.

Reply:
Fixed